app/controllers/concerns/saml_idp_auth_concern.rb

```diff
@@ -22,13 +21,6 @@ module SamlIdpAuthConcern
 
   private
 
-  def block_biometric_requests_in_production
-    if @saml_request_validator.biometric_comparison_requested? &&
-       !FeatureManagement.idv_allow_selfie_check?
-      render_not_acceptable
-    end
-  end
```

Comment:
I grumbled about this on a pairing call, and it's not totally relevant here, but note that the "in production" part is confusingly-named. AFAICT it just referred to the feature flag (which was initially off in prod); it didn't actually check if were were in prod per se.

Reply:
At one point this did explicitly check if it was in production. That was removed but the name was not updated.